brake_app/flight_data.py

```diff
@@ -1,7 +1,7 @@
 import pandas as pd
 
 class FlightData:
-    DATA_TO_COPY = ["ax", "ay", "az", "gx", "gy", "gz", "pressure", "altitude"]
+    DATA_TO_COPY = ["ax", "ay", "az", "gx", "gy", "gz", "pressure", "tempreature"]
```

Comment:
temperature ://

Reply:
wiem ale w csv tak miałem to zmieniłem w pythonie xDDD